cmd/cortex/main.go

```diff
@@ -313,6 +313,10 @@ func setupIngester(
 	}
 	prometheus.MustRegister(ingester)
 
+	router.Path("/push").Handler(http.HandlerFunc(ingester.PushHandler))
+	router.Path("/query").Handler(http.HandlerFunc(ingester.QueryHandler))
+	router.Path("/label_values").Handler(http.HandlerFunc(ingester.LabelValuesHandler))
+	router.Path("/user_stats").Handler(http.HandlerFunc(ingester.UserStatsHandler))
```

Comment:
Maybe put comments around these to indicate they are there temporarily while we upgrade?